src/main/java/seedu/mtracker/console/AddInstrumentParser.java

```diff
@@ -13,12 +13,16 @@
 
     public static final int INSTRUMENT_COMMAND_INDEX = 0;
 
-    protected static final ArrayList<String> parameters = new ArrayList<>();
+    protected static ArrayList<String> parameters;
 
     public ArrayList<String> getParameters() {
         return parameters;
     }
 
+    public void setParameter() {

```

Comment:
naming should be consistent to setParameters

Reply:
also v similar naming to setParams in the command class maybe could change this name ?